Examples/Tests/collider_relevant_diags/inputs_3d_multiple_particles

```diff
@@ -0,0 +1,137 @@
+my_constants.ny = 8
+my_constants.nz = 8
+
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 1
+amr.n_cell = nx ny nz
+warpx.numprocs = 1 1 1
+amr.max_level = 0
+geometry.dims = 3
+geometry.prob_lo = 0 0 0
+geometry.prob_hi =  8 8 8
+particles.do_tiling = 0
+warpx.use_filter = 0
+
+#################################
+######## BOUNDARY CONDITION #####
+#################################
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
+boundary.particle_lo = periodic periodic periodic
+boundary.particle_hi = periodic periodic periodic
+
+#################################
+############ NUMERICS ###########
+#################################
+algo.maxwell_solver = ckc
+warpx.cfl = 0.99
+algo.particle_shape = 1
+
+#################################
+############ FIELDS #############
+#################################
+particles.E_ext_particle_init_style = constant
+particles.B_ext_particle_init_style = constant
+particles.E_external_particle = 10000. 0. 0.
+particles.B_external_particle = 0. 5000. 0.
+
+#################################
+########### PARTICLES ###########
+#################################
+particles.species_names = beam_p beam_e pho
+particles.photon_species = pho
+
+beam_e.species_type = electron
+beam_e.injection_style = MultipleParticles
+beam_e.multiple_particles_pos_x = 4.5  3.5  0.5
+beam_e.multiple_particles_pos_y = 4.5  2.5  1.5
+beam_e.multiple_particles_pos_z = 4.5  1.5  1.5
+beam_e.multiple_particles_ux = 0.3 0.2 0.1
+beam_e.multiple_particles_uy = 0.4 -0.3 -0.1
+beam_e.multiple_particles_uz = 0.3 0.1 -10.
+beam_e.multiple_particles_weight = 1. 2 3
+beam_e.initialize_self_fields = 0
+beam_e.self_fields_required_precision = 5e-10
+beam_e.do_qed_quantum_sync = 1
+beam_e.qed_quantum_sync_phot_product_species = pho
+beam_e.do_not_push = 1
+beam_e.do_not_deposit = 1
+
+beam_p.species_type = positron
+beam_p.injection_style = MultipleParticles
+beam_p.multiple_particles_pos_x = 4.5 3.5 0.5
+beam_p.multiple_particles_pos_y = 4.5 2.5 1.5
+beam_p.multiple_particles_pos_z = 4.5 1.5 1.5
+beam_p.multiple_particles_ux = 0.3 0.2 0.1
+beam_p.multiple_particles_uy = 0.4 -0.3 -0.1
+beam_p.multiple_particles_uz = 0.3 0.1 -10.
+beam_p.multiple_particles_weight = 1. 2 3
+beam_p.initialize_self_fields = 0
+beam_p.self_fields_required_precision = 5e-10
+beam_p.do_qed_quantum_sync = 1
+beam_p.qed_quantum_sync_phot_product_species = pho
+beam_p.do_not_push = 1
+beam_p.do_not_deposit = 1
+
+pho.species_type = photon
+pho.injection_style = none
+
+#################################
+############# QED ###############
+#################################
+qed_qs.photon_creation_energy_threshold = 0.
+qed_qs.lookup_table_mode = builtin
+qed_qs.chi_min = 1.e-3
+warpx.do_qed_schwinger = 0
+
+#################################
+######### DIAGNOSTICS ###########
+#################################
+# FULL
+diagnostics.diags_names = diag1 diag2
```

Comment:
Maybe for clarity we could call these something like `diag_yt` and `diag_opmd`

Reply:
I think that the CI machine uses `diag1` as a default name to compare the plotfiles with the checksums, right @EZoni?

Reply:
Yes, I think the `diag1` name is hard-coded here:
https://github.com/ECP-WarpX/WarpX/blob/cf3b8a9258fa723a109b46c8154b836f10287d01/Regression/WarpX-tests.ini#L51-L53